hervebutera/my_brand-backend
# Ft contact queries

#### What does this PR do?
This PR adds API endpoints that allow sending a query to the website owner and
adds API endpoint that allow website owner to retrieve queries
#### Description of Task to be completed?
Adds API endpoint for sending queries to the website owner.
Adds API endpoint for retrieving the queries available
The posting API has validation feature to pass for a query to be posted to database
####How should this be manually tested?
Clone the repository and install all the necessary packages as shown by the package.json file,
then run npm start.
Also use postman to check if they pass.
#### Any background context you want to provide?
Validation before posting the query to database was included. 

Screenshots (if appropriate)
<img width="796" alt="Capture2" src="https://user-images.githubusercontent.com/48241489/97109461-09ebda00-16dc-11eb-9d65-797b4fe0c2cb.PNG">
<img width="799" alt="Capture" src="https://user-images.githubusercontent.com/48241489/97109463-0ce6ca80-16dc-11eb-873b-a02a2d928d7b.PNG">
<img width="798" alt="Capture1" src="https://user-images.githubusercontent.com/48241489/97109464-0d7f6100-16dc-11eb-9e84-695d8883c0e5.PNG">
